src/control/server/mgmt_pool.go

```diff
@@ -124,41 +125,57 @@ func (svc *mgmtSvc) calculateCreateStorage(req *mgmtpb.PoolCreateReq) error {
 	if len(req.GetRanks()) == 0 {
 		return errors.New("zero ranks in calculateCreateStorage()")
 	}
-	if req.GetScmratio() == 0 {
-		req.Scmratio = DefaultPoolScmRatio
+
+	// the engine will accept only 2 tiers - add missing
+	if len(req.GetTierratio()) == 0 {
+		req.Tierratio = []float64{DefaultPoolScmRatio, DefaultPoolNvmeRatio}
+	} else if len(req.GetTierratio()) == 1 {
+		req.Tierratio = append(req.Tierratio, 0)
 	}
 
 	storagePerRank := func(total uint64) uint64 {
 		return total / uint64(len(req.GetRanks()))
 	}
 
+	if len(req.Tierbytes) == 0 {
+		req.Tierbytes = make([]uint64, len(req.Tierratio))
```

Comment:
why are these not named "req.TierBytes" and "req.TierRatio" ?

Reply:
This is to follow the naming of other existing fields in the struct, like: Usergroup, Numsvcreps, Totalbytes, Numranks...

Reply:
yes this is technical debt in the Proto definitions for some of the request structures like for pool create request, it should be addressed outside of this PR: https://jira.hpdd.intel.com/browse/DAOS-8014